### PR TITLE
Update LICENSE.md

### PR DESCRIPTION
**Pull request by siwelwerd**
Copied over language about assigning copyright

### CODE REVIEW

**Comment by StevenClontz** [comment]
This language is in https://github.com/TeamBasedInquiryLearning/library/blob/main/CONTRIBUTING.md already, but we should make it more discoverable...

File: LICENSE.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Copyright and Licensing
 
-All contents of this repository are copyright 2017-2024 Steven Clontz and Drew Lewis,
-and are licensed for free public use under the following licenses:
+All contents of this repository are copyright 2017-2024 Steven Clontz and Drew Lewis. By contributing to this repository, you agree that you are providing the copyright owners an irrevocable, perpetual, non-exclusive, fully-paid, worldwide license, with the right to sublicense through multiple tiers, to use, distribute, reproduce, modify, adapt, publish, translate, publicly perform, and publicly display or otherwise use your contributions (in whole or in part) in any format or medium now known or later developed, without compensation or notification to or permission from the user of any kind.
+
+All contents of this repository are licensed for free public use under the following licenses:
```
Comment (StevenClontz) [inline LICENSE.md:5]:
```suggestion
All contents of this repository are copyright 2017-2024 Steven Clontz and Drew Lewis.
and are licensed for free public use under the following licenses.
[Contributions](https://github.com/TeamBasedInquiryLearning/library/blob/main/CONTRIBUTING.md) are warmly welcomed and encouraged.
```

**Comment by siwelwerd** [comment]
Is there a Github setting to make CONTRIBUTING.md show up like the License and Readme on the landing page?  I think it's fair to assume that if I overlooked that file, other contributors may as well.

File: LICENSE.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # Copyright and Licensing
 
-All contents of this repository are copyright 2017-2024 Steven Clontz and Drew Lewis,
-and are licensed for free public use under the following licenses:
+All contents of this repository are copyright 2017-2024 Steven Clontz and Drew Lewis.
+and are licensed for free public use under the following licenses.
+[Contributions](https://github.com/TeamBasedInquiryLearning/library/blob/main/CONTRIBUTING.md) are warmly welcomed and encouraged.
```
Comment (StevenClontz) [inline LICENSE.md:5]:
```suggestion
Contributions are warmly welcomed and encouraged; see [CONTRIBUTING.md](CONTRIBUTING.md) for details.
```

**Comment by StevenClontz** [comment]
> Is there a Github setting to make CONTRIBUTING.md show up like the License and Readme on the landing page? I think it's fair to assume that if I overlooked that file, other contributors may as well.

CONTRIBUTING.md is the standard file for this; I think it shows up when you open issues or make pull requests if you're not an admin. Agreed we should enable whatever we need to enable to make this visible.

**Comment by StevenClontz** [comment]
image

maybe also PRs but I'm not a first-time PR creator here...

File: LICENSE.md (hunk 2 above)
Comment (StevenClontz) [reply LICENSE.md]: @siwelwerd consider this further tweak 👆

Comment (StevenClontz) [reply LICENSE.md]: oh you did, weird, it didn't close this. ah well

**Review by StevenClontz — APPROVED** [review]

**Comment by siwelwerd** [comment]
Yeah, looks like it should show up for issues and PRs for everyone, according to the [Github Docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#about-contributing-guidelines)

**Pull request merged**